versions/1.0.0.md

```diff
@@ -111,10 +111,11 @@ This object represents one or more changes to be applied to the target document
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="actionTarget"></a>target | `string` | **REQUIRED** A JSONPath query expression referencing the target objects in the target document.
```

Comment:
I noticed that anchors are removed.  I think they don't quite work well with GitHub's formatting, anyway, but it's worth pointing out. Not sure if this is used with other tooling.  Not a blocker.